image-counter-lambda/src/getCredentials.ts

```diff
@@ -0,0 +1,17 @@
+import AWS from "aws-sdk";
+
+async function getCredentials() {
+  const s3 = new AWS.S3();
+  const params = {
+    Bucket: "grid-conf",
+    Key: `${process.env.STAGE}/image-counter-lambda/conf.json`
+  };
+
+  const data = await s3.getObject(params).promise();
+  const buffer = data.Body;
+  const body = buffer ? buffer.toString("utf8") : "";
```

Comment:
NB: bonus new Typescript 3.7 feature -- in place of the ternary expression we should be able to write `buffer.toString?.("utf8") || ""` [thanks to optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) 😁